### DISCUSSION TITLE
使用 Tradytics 进行日内期权交易 - forecho's Blog

### DISCUSSION THREAD

**Discussion by giscus** [forecho/hugo-blog#35] [Announcements]
# 使用 Tradytics 进行日内期权交易 - forecho's Blog

引言
前一段时间和几个朋友以一个非常低的价格拼车买了 Tradytics 的会员，一有时间我就研究如何使用，虽然 Tradytics 太复杂了，但是有很多有用的数据，可以帮我们看清目前的市场状况，然后做出正确的交易决策。
我会把我学到的东西分享出来，希望能帮助到大家。系列文章点击 Tradytics 查看。
Tradytics 是什么？
Tradytics 是一个人工智能驱动的交易平台，提供了各种工具来帮助我们在市场上获得优势，它每天分析超过一百万个数据点，来自期权流量、股票日内数据、新闻、内部交易等。
基于这些分析，通过专有人工智能算法，提取有用信息，帮助我们提前评估市场走向。

https://blog.forecho.com/day-trading-options-with-tradytics.html

**Comment by imoreapps** [comment]
朋友，有没有合租的tradytics账号，想合租一个进去学习学习😁

**Reply by forecho** [reply]
https://t.me/forecho 联系我
